content/learn/quick-start/introduction/_index.md

```diff
@@ -38,6 +38,6 @@ Bevy is still in the early stages of development. Important features are missing
 
 If you are currently trying to pick an engine for your Next Big Project™, we recommend that you check out [Godot Engine](https://godotengine.org). It is currently much more feature-complete and stable. And it is also free, open-source, and [scriptable with Rust](https://github.com/godot-rust/gdext)!
```

Comment:
there's a `learn/book/contributing` link on line 29 that needs to be updated in this file